### PR TITLE
Machine view base

### PR DESCRIPTION
**Pull request by huwshimi**
Create a base machine view component with a list of unplaced services.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- view the gui with `/:flags:/mv/`
- open the machine view
- add a service
- you should see some onboarding
- an unplaced unit should appear
- click 'Auto place'
- new onboarding should appear and the count next to "sandbox" should be updated

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3711/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3712/
Test PASSed.

File: jujugui/static/gui/src/app/components/machine-view/header/header.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('machine-view-header', function() {
+
+  juju.components.MachineViewHeader = React.createClass({
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/machine-view/header/header.js]: Even though this is repeated multiple times I'm not sure having it as a separate component really gains anything. Is there going to be some additional logic here in the future?

**Comment by hatched** [comment]
:+1: This looks like a great start. I had a comment on the header component previously but then I remembered that there is going to have to be quite a bit of logic in there going forward.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3720/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3721/
Test PASSed.

**Comment by huwshimi** [comment]
As discussed, shipping to unblock :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**